assets/js/media.js

```diff
@@ -0,0 +1,16 @@
+
+let btnLoad = document.getElementById("loadMedia");
+let btnHide = document.getElementById("hideMedia");
+let medias = document.getElementById('medias');
```

Comment:
Codacy has a fix for the issue: [Replace `'medias'` with `"medias"`](https://app.codacy.com/gh/Florimond-Jouffroy/Openclassrooms-Project-006/pullRequest?prid=9544790)
```suggestion
let medias = document.getElementById("medias");
```